runtime/khala/src/lib.rs

```diff
@@ -825,14 +825,18 @@ parameter_types! {
     pub KhalaTreasuryAccount: AccountId = TreasuryPalletId::get().into_account();
     pub CheckingAccount: AccountId = PalletId(*b"checking").into_account();
 }
-pub type Barrier = (
-    TakeWeightCredit,
-    AllowTopLevelPaidExecutionFrom<Everything>,
-    // Expected responses are OK.
-    AllowKnownQueryResponses<PolkadotXcm>,
-    // Subscriptions for version tracking are OK.
-    AllowSubscriptionsFrom<Everything>,
-);
+
+pub type Barrier = xcm_config::DenyThenTry<
+    xcm_config::DenyReserveTransferToRelayChain,
```

Comment:
I believe this will disable KSM/DOT transfer. Looks not good.

Reply:
It _should_ just prevent doing a reserve transfer of PHA with Khala as the reserve, but not of using the Relay Chain as a KSM reserve like it should. @gilescope should know.

Reply:
With this barrier set, we can not send KSM/DOT back to relaychain and also can not send assets reserved on our chain to relaychain. I think it's ok because no KSM on khala so far, we don't need to transfer it back. But technically, I don't see any reason why we should disable transfer KSM/DOT back to relaychain. Perhaps @gilescope can post more information

Reply:
Discussed with @tolak , we decide to revert this change, thanks you @joepetrowski for mention this!